stl/inc/format

```diff
@@ -1187,6 +1187,24 @@ public:
         }
     }
 
+    _NODISCARD size_t _Calculate_reserve_size() const noexcept {

```

Comment:
Naming: Is "reserve size" potentially confusing? I know what it's trying to say, but `reserve()` affects `capacity()`, not `size()`.

Additionally, is this an exact calculation? It appears to be an estimate (partially because `_Fmt.size() +` outside isn't taking into account any specifiers, partially because I think there are width/precision/etc. options that could affect the output).

Reply:
Renamed to `_Estimate_required_capacity`